frontend/src/components/opportunity/OpportunityDownload.tsx

```diff
@@ -1,37 +1,21 @@
 "use client";
 
-import { useTranslations } from "next-intl";
-import { Button, Link } from "@trussworks/react-uswds";
+import { OpportunityDocument } from "src/types/opportunity/opportunityResponseTypes";
 
-import { USWDSIcon } from "src/components/USWDSIcon";
+import { useTranslations } from "next-intl";
+import { Link } from "@trussworks/react-uswds";
 
 type OpportunityDownloadProps = {
-  nofoPath: string;
-  opportunityId: number;
+  attachments: OpportunityDocument[];
 };
 
-const downloadNOFO = (nofoPath: string) => {
-  window.open(nofoPath, "_blank");
-};
-
-const OpportunityDownload = ({
-  nofoPath,
-  opportunityId,
-}: OpportunityDownloadProps) => {
+const OpportunityDownload = ({ attachments }: OpportunityDownloadProps) => {
   const t = useTranslations("OpportunityListing.description");
 
   return (
     <>
-      {nofoPath.length > 0 ? (
+      {attachments.length > 0 ? (
```

Comment:
(@doug-s-nava don't let this prevent merging. I just wanted to raise a Q for design.)

@crystabelrangel, should the "jump to" link be disabled instead of nonexistent when there are no attachments? I'm wondering what the preferred UI would be. Is it more confusing for the link to not be there or not be interactive? Should we consider another way to indicate at a glance that there are no documents?